src/main/java/com/trailfinder/dto/ProfileDTO.java

```diff
@@ -8,12 +8,28 @@
  */
 public class ProfileDTO {
 
-	// Private Fields
 	private int profileId;
 	private String firstName;
 	private String bio;
 	private List<EventDTO> events;
 	
+	public ProfileDTO() {	
+	}
+	
+	/**
+	 * Create and initialize a new instance of the ProfileDTO
+	 * @param profileId
+	 * @param firstName
+	 * @param bio
+	 * @param events
+	 */
+	public ProfileDTO(int profileId, String firstName, String bio, List<EventDTO> events) {
```

Comment:
Created a constructor that makes initializing the DTO simpler

Reply:
Excellent Javadoc... add a bit of description to each of the params to make it more valuable.